code/go/0chain.net/smartcontract/storagesc/allocation_test.go

```diff
@@ -1654,9 +1654,13 @@ func TestStorageSmartContract_closeAllocation(t *testing.T) {
 	_, err = ssc.closeAllocation(&tx, alloc, balances)
 	requireErrMsg(t, err, errMsg1)
 
+	var conf = Config{
+		MaxChallengeCompletionTime: 30 * time.Minute,
+	}
+
 	// 2. close (all related pools has created)
 	alloc.Expiration = tx.CreationDate +
-		toSeconds(alloc.ChallengeCompletionTime) + 20
+		toSeconds(conf.MaxChallengeCompletionTime) + 20
```

Comment:
You can always call getConfig diredctly, it should be in the MPT. Alternately use the same constant to set it here and in [createNewTestAllocation](https://github.com/0chain/0chain/blob/d7321ef7a912eac8763e70eb6de034f69de6fadf/code/go/0chain.net/smartcontract/storagesc/allocation_test.go#L1512).

Reply:
Calling getConfg directly gives the default value, that fails the test